exo-go/src/template_helpers/index.go

```diff
@@ -167,3 +168,49 @@ func CreateTmpServiceDir(chosenTemplate string) string {
 	}
 	return serviceTmpDir
 }
+
+// AddTemplate fetches a remote template from GitHub and stores it
+// under templateDir, returns an error if any
+func AddTemplate(gitURL, templateName, templateDir string) error {
+	if output, err := processHelpers.Run("", "git", "submodule", "add", "--name", templateName, gitURL, templateDir); err != nil {
```

Comment:
Thoughts on making the API of Run / Start back to `(command, dir string)`? (thats what it was in the test helpers when it used shellwords internally). Why was it changed?

I don't like that the dir looks as if its part of the command. I'd prefer the command either be a string or an array of strings. (instead of multiple string arguments)